data.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.0;
```

Comment:
Is this a semantic change?

Reply:
I started using this new pragma because I saw it on the docs. It does not make differece right now, but seems more complete.

Reply:
This is how solidity style guide suggests https://solidity.readthedocs.io/en/v0.5.1/style-guide.html